config/site_settings.yml

```diff
@@ -170,6 +170,16 @@ basic:
      - facebook
      - google+
      - email
+  share_links_urlencode:
```

Comment:
You need to add a translation in `server.en.yml` when you add a new site setting.